.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,65 @@
+---
+  name: Create JIRA ticket for dependabot prs
+
+  on: pull_request
+
+  permissions: 
+    pull-requests: write
+    contents: read
+  jobs:
+    jira_task:
+      name: Create Jira issue
+      runs-on: ubuntu-latest
+      if: github.actor == 'dependabot[bot]'
+      steps:
+      - name: Create JIRA ticket
+        id: create
+        shell: bash
+        env:
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          PR_URL: ${{ github.event.pull_request.html_url }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+          JIRA_ASSIGNEE: ${{ secrets.ASSIGNEE_JIRA_TICKET }}
+        run: |
+          ASSIGNED_TEAM=$(git diff HEAD~1..HEAD -- go.mod | grep -v "// indirect" | grep -i "^\-" | grep -v "^\-\-\-" | awk '{print $2}' | xargs -I $ echo "jq -r \".\\\"$\\\"\" < build/ci/library_owners.json" | sh | xargs -I $ echo "jq -r \".\\\"$\\\"\" < build/ci/library_owners_jira.json" | sh | head -1)
```

Comment:
the teams in that file are github teams and not jira teams

Reply:
that is why I've created `library_owners_jira.json`

Reply:
never mind I see we added a mapping